### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by huynq0911**
**Which issue(s) this PR fixes**:
N/A

**What this PR does / why we need it**:
Add pull request template

**Docs Changes**:
.github/PULL_REQUEST_TEMPLATE.md

**Release Note**:
N/A

### CODE REVIEW

**Comment by jtattermusch** [comment]
I'm not sure how common it is to use pull request template in other github projects (as opposed to using issue template, which is quite common).

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!--
+Thank you for contributing to GRPC!
+Please provide the following information to help us make the most of your pull request:
```
Comment (jtattermusch) [inline .github/PULL_REQUEST_TEMPLATE.md]: there should be a link to https://github.com/grpc/grpc/blob/master/CONTRIBUTING.md

**Comment by huynq0911** [comment]
> I'm not sure how common it is to use pull request template in other github projects (as opposed to using issue template, which is quite common).

As my knowledge from other github projects, sometime there are only some PR without Issue . So I think this template will help some contributor to make PR only.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (huynq0911) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes. Thank for suggesting!

**Pull request closed without merging**